client/me/purchases/list/site/style.scss

```diff
@@ -1,3 +1,22 @@
 .purchases-site {
 	margin-bottom: 15px;
+
+	.section-header {
+		@include breakpoint( "<660px" ) {
+			display: block;
+		}
+	}
+}
+
+.purchases-site__slug {
+	color: darken( $gray, 15% );
+	font-size: 12px;
+	line-height: 28px;
+}
+
+.section-header__label,
+.purchases-site__slug {
+	@include breakpoint( "<660px" ) {
+		line-height: 140%;
```

Comment:
Why set the line-height in `%`? Not really harming anything just seems better to keep your units consistent between brreakpoints.